### PR TITLE
Added moving average filter

### PR DESCRIPTION
**Pull request by DaanGro**
We have to discuss wether to implement this in get_match() or leave it to the user.
Also for different filters, poetry has issues regarding adding scipy (clashes with our current Python versions), we have to discuss how to handle this.

### CODE REVIEW

**Comment by DaanGro** [comment]
Added scipy, and savitzky golay filter

File: databallpy/filters.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+def filter_data(
+    df: pd.DataFrame,
+    input_columns: list,
+    kind: str,
+    window_length: int = 5,
+    poly_order: int = 2,
+) -> pd.DataFrame:
+    """Function to filter specified columns, available filters are: moving average
```
Comment (Alek050) [inline databallpy/filters.py:12]: Add savgol_filter

File: databallpy/filters.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+from scipy.signal import savgol_filter
+
+
+def filter_data(
+    df: pd.DataFrame,
+    input_columns: list,
+    kind: str,
```
Comment (Alek050) [inline databallpy/filters.py:8]: Maybe refactor this to `filter_type` or something. I think `kind` is not specific enough.

**Review by Alek050 — CHANGES_REQUESTED** [review]

**Comment by Alek050** [comment]
> We have to discuss wether to implement this in get_match() or leave it to the user. Also for different filters, poetry has issues regarding adding scipy (clashes with our current Python versions), we have to discuss how to handle this.

I think we have to leave this to users for most of the part. The package needs to be able to read in all data and support features, but it is up to the user how data is preprocessed and features are calculated.

**Review by Alek050 — APPROVED** [review]

**Pull request merged**